### PR TITLE
Dominator Shipping

### LINKED ISSUE #11297: You can ship the dominator and it still works.
I shouldent need to explain why this is terible

### PR DESCRIPTION
**Pull request by Cheridan**
Do you have a permit for that? Fixes #11297

### CODE REVIEW

**Comment by optimumtact** [comment]
>blacklist

**Comment by Anonus** [comment]
>whitelisting everything

**Comment by WJohn** [comment]
I feel like this could be done with the white ship, if it isn't already blacklisted.

**Comment by Ikarrus** [comment]
The dominator shouldnt even be able to be spawned on the cargo shuttle. Or any shuttle. 

A better solution to this is to check for shuttle areas when spawning a dominator from a gangtool

**Comment by MrPerson** [comment]
nukexdominator otp

File: code/controllers/subsystem/shuttles/supply.dm [hunk 1]
```diff
@@ -216,6 +216,7 @@
 		/obj/item/device/radio/beacon,
 		/obj/machinery/the_singularitygen,
 		/obj/singularity,
```
Comment (duncathan) [inline code/controllers/subsystem/shuttles/supply.dm]: Well that's just boring

Shut up I know this isn't part of the pr

**Comment by Cheridan** [comment]
@Ikarrus Should dominators work off Z1 in the first place? It would be consistent with most of our other game modes and fix any other potential issues/exploits

File: code/controllers/subsystem/shuttles/supply.dm (hunk 1 above)
Comment (Firecage) [inline code/controllers/subsystem/shuttles/supply.dm]: ;)

**Comment by Alek2ander** [comment]
You should only be able to spawn a dominator in an area tagged for your gang. That automatically blacklists all bad places it can be.

File: code/controllers/subsystem/shuttles/supply.dm (hunk 1 above)
Comment (ike709) [inline code/controllers/subsystem/shuttles/supply.dm]: CentCom filed a complaint after some cargo techs created a singularity in the cargo bay and managed to get it into the shuttle and send it away.

**Comment by Ikarrus** [comment]
@Cheridan Check for the shuttle /area/

IIRC We still use them, right?

**Comment by optimumtact** [comment]
Shuttle areas are not guaranteed to be present, especially if someone is doing a custom shuttle event

**Comment by Cheridan** [comment]
I'll try @Alek2ander's idea then

**Comment by Ikarrus** [comment]
The downside to @Alek2ander 's proposed method is that it would also blacklist maintenance areas, which were intended to be left usable for a dom base.

Might not be the worst thing to happen, but it's worth mentioning.

**Comment by Razharas** [comment]
Hows the progress @Cheridan

**Comment by Razharas** [comment]
@Cheridan ?

**Comment by Dorsisdwarf** [comment]
@-@-@-@Cheridan -breaker!

**Pull request closed without merging**